build.sbt

```diff
@@ -92,7 +92,8 @@ val commonSettings =
         "-unchecked",
         "-deprecation",
         "-encoding", "utf8",
-        "-Xfatal-warnings",
+        // TODO: Turn it on back when we break compatibility with Flink 1.6: see comments in StoppableExecutionEnvironment.prepareMiniClusterResource
+//        "-Xfatal-warnings",
```

Comment:
I guess you could probably turn `Xfatal-warnings` back on using https://github.com/ghik/silencer
also related: https://github.com/ghik/silencer#message-pattern